app.py

```diff
@@ -48,4 +48,5 @@ def forecast(self):
 # location.name, .region, .country, .localtime
 # current.temp_f, .condition.text
 # forecast.forecastday[0], forecast.forecastday[0].maxtemp_f, forecast.forecastday[0].mintemp_f
-# forecast.forecastday[1], forecast.forecastday[1].maxtemp_f, forecast.forecastday[1].mintemp_f, forecast.forecastday[1].condition.text
\ No newline at end of file
+# forecast.forecastday[1], forecast.forecastday[1].maxtemp_f, forecast.forecastday[1].mintemp_f, forecast.forecastday[1].condition.text
+#changes
```

Comment:
Load the API key from an environment variable, so it's secure (we don't want people stealing your key.

Reply:
refactor you script to follow the if __name__ == "__main__" python code execution format ([more details](https://realpython.com/python-main-function/)).